From b3fcf04656319c1210b32ee046c4d94c6abacfff Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 30 Jul 2017 13:32:37 +0900
Subject: [PATCH 1/3] submission removes Received header

If using submission port with the account based authentication, let's not
expose unnecessary internal info of the local exim running on the desktop
machine.  Let's keep the mail header sent to the submission host to be just
like what happens with Thunderbird MUA connecting to the submission port.
---
 debian/debconf/conf.d/transport/30_exim4-config_remote_smtp_submission | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/debconf/conf.d/transport/30_exim4-config_remote_smtp_submission b/debian/debconf/conf.d/transport/30_exim4-config_remote_smtp_submission
index 5b485c8..f019e17 100644
--- a/debian/debconf/conf.d/transport/30_exim4-config_remote_smtp_submission
+++ b/debian/debconf/conf.d/transport/30_exim4-config_remote_smtp_submission
@@ -8,6 +8,7 @@ remote_smtp_submission:
   port = 587
   hosts_require_auth = *
   hosts_require_tls = *
+  headers_remove = Received
 .ifdef REMOTE_SMTP_HEADERS_REWRITE
   headers_rewrite = REMOTE_SMTP_HEADERS_REWRITE
 .endif
-- 
2.11.0

